src/components/Contact/contactinfo.tsx

```diff
@@ -112,6 +118,23 @@ function ContactInfo({ contact, image }: ContactInfoProps) {
         </label>
         <span className='text-xl'> {contact.notes} </span>
       </Box>
+      <div className='mb-2'>
+        <button
+          type='button'
+          onClick={() => {
+            setAlert({
+              variant: AlertVariant.security,
+              title: 'Delete Contact',
+              text: 'Are you sure?',
+              position: 'bottom',
+              confirmFunction: () => DeleteContact
```

Comment:
This should be confirmFunction: DeleteContact. Otherwise when confirmFunction is run, it will return DeleteContact but won't run it.